survlime/survlime_explainer.py

```diff
@@ -264,4 +274,68 @@ def solve_opt_problem(
         prob = cp.Problem(objective)
         result = prob.solve(verbose=verbose)
         cox_coefficients = b.value[:, 0]
+
+        self.computed_weights = cox_coefficients
         return cox_coefficients
+
+    def plot_weights(self, figsize: Tuple[int, int] = (10, 10),
+                     feature_names: List[str] = None,
+                     scale_with_data_point: bool = False) -> None:
+        """Plot the weights of the computed model using 
+            seaborn as plotting library
+        Args:
+            figsize (Tuple[int, int]): size of the figure
+        """
+        if self.computed_weights is None:
```

Comment:
If you call this function only when the user specifies it in explain_instance method, this is not needed